ValheimExportHelper/FixShaders.cs

```diff
@@ -23,14 +23,16 @@ public override void Export()
 
     private void CreateScriptsDir()
     {
-      ScriptsDir = Path.Join(CurrentRipper.Settings.AssetsPath, "Scripts");
+      ScriptsDir = Path.Join(CurrentRipper.Settings.AssetsPath, "Editor");
       Directory.CreateDirectory(ScriptsDir);
     }
 
     private void AddEditorScript()
     {
       File.WriteAllText(Path.Join(ScriptsDir, "Editor.cs"), Resource.Editor);

```

Comment:
I meant to remove the line `File.WriteAllText(Path.Join(ScriptsDir, "Editor.cs"), Resource.Editor);`, it's also in `AddEditorAssets.cs`.

Reply:
The "unity" thing is that scripts in the "Editor" folder (even if Editor is nested inside Scripts or another folder) as long as this folder is merely called "Editor" is compiled / ran only in editor env.

Reply:
in essence avoids all together having to tag #if UNITY_EDITOR as putting the scripts in the editor folder tags them for editor use only